### PR TITLE
fix: honor getViteConfig inlineAstroConfig.logLevel setting

### PR DESCRIPTION
**Pull request by spacedawwwg**
## Changes

Use `inlineAstroConfig.logLevel` if passed to `getViteConfig`

## Testing

Tested locally using fork.

## Docs

There are no additional docs as this change fixes what should already happen.

### CODE REVIEW

File: packages/astro/src/config/index.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export function getViteConfig(
 		]);
 		const logger = new Logger({
 			dest: nodeLogDestination,
-			level: 'info',
+			level: inlineAstroConfig.logLevel ?? 'info'
```
Comment (bluwy) [inline packages/astro/src/config/index.ts:38]: Can we create the logger by importing `createNodeLogger` instead? That should be even accurate and we can create the logger like `const logger = createNodeLogger(inlineAstroConfig)`

https://github.com/withastro/astro/blob/c480b8f2c4a6282ccee706e9121ceacd91f5172b/packages/astro/src/core/config/logging.ts#L5-L12

File: .changeset/cuddly-feet-float.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+fix: honor getViteConfig inlineAstroConfig.logLevel setting
```
Comment (bluwy) [inline .changeset/cuddly-feet-float.md:5]: nit: we usually follow a format like this
```suggestion
Honors `inlineAstroConfig` parameter in `getViteConfig` when creating a logger
```

**Comment by spacedawwwg** [comment]
@bluwy changes made 👍🏻

**Review by bluwy — APPROVED** [review]
Looks great!

**Review by ematipico — APPROVED** [review]

**Pull request merged**